docs-src/go/generated/inspect-the-new-event-history.md

```diff
@@ -0,0 +1,53 @@
+---
+id: inspect-the-new-event-history
+title: Inspect the new Event History
+sidebar_label: Inspect new Event History
+description: After making changes to the code, we must update the Event History JSON file to get tests to pass.
+tags:
+- tests
+- replay
+- event history
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-go/blob/port_replay_test_dacx/backgroundcheck_replay/workflows/backgroundcheck.go. -->
+
+After updating your Workflow code to include the logging and Timer, run your tests again.
+You should expect to see the `TestReplayWorkflowHistoryFromFile` test fail.
+This is because the code we added creates new Events and alters the Event History sequence.
+
+To get this test to pass, we must get an updated Event History JSON file.
+[Start a new Workflow](/go/chapter-project-setup/backgroundcheck-boilerplate-start-workflow) and after it is complete [download the Event History as a JSON object](/go/chapter-durable-execution/retrieve-event-history).
+
+:::info Double check Task Queue names
+
+Reminder that this guide jumps between several sample applications using multiple Task Queues.
+Make sure you are starting Workflows on the same Task Queue that the Worker is listening to.
+And, always make sure that all Workers listening to the same Task Queue are registered with the same Workflows and Activities.
+
+:::
+
+If you inspect the new Event History, you will see two new Events in response to the `workflow.Sleep()` API call which send the [StartTimer Command](/references/commands#starttimer) to the Server:
+
+- `TimerStarted`
+- `TimerFired`
+
+However, it is also important to note that you don't see any Events related to logging.
+And if you were to remove the Sleep call from the code, there wouldn't be a compatability issue with the previous code.
+This is to highlight that only certain code changes within Workflow code is non-deterministic.
+The basic thing to remember is that if the API call causes a [Command](/references/commands) that causes Events, then it is a non-deterministic change.
```

Comment:
```suggestion
The basic thing to remember is that if the API call causes a [Command](/references/commands) to create an Events in the Workflow History that takes a new path from previous Workflow Event History, then it is a non-deterministic change.
```

This needs some rephrasing, but an Event in the Workflow History isn't non-deterministic. 
A change from one Workflow History to another, that takes a new code path, is non-deterministic.